tests/operators/test_local.py

```diff
@@ -646,6 +731,8 @@ def test_dbt_base_operator_no_partial_parse() -> None:
     assert "--no-partial-parse" in cmd
 
 
+# TODO: Make test compatible with Airflow 3.0. Issue:https://github.com/astronomer/astronomer-cosmos/issues/1705
+@pytest.mark.skipif(version.parse(airflow_version).major == 3, reason="Test need to be updated for Airflow 3.0")
```

Comment:
Same question if this test has started failing now?

Reply:
Nice catch, @pankajkoti , I think it may have just been a rebasing issue - removing now

Reply:
Addressed in 2a113a5